lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -101,18 +105,21 @@ sub USAGE() is export {
 }
 
 
-sub dump-model (IO::Path $path) is export {
+sub dump-model (IO::Path $path, $variants) is export {
     die "ERROR: dump expects a .nhd file" unless $path.extension eq 'nhd';
 
     my $module-path = $path.parent;
     my $module-file = $path.basename;
     my $module      = $path.extension('').basename;
 
-    my $model = timed "load $module", { load-model-using-cache($*HOME.add('.agrammon'), $module-path, $module) };
+    my $model = timed "load $module", {
+        my %preprocessor-options = set($variants.split(","));
+       load-model-using-cache($*HOME.add('.agrammon'), $module-path, $module, %preprocessor-options)
```

Comment:
Something looks a space off with the indentation here

Reply:
Fixed.